src/HDF5.jl

```diff
@@ -1424,7 +1424,7 @@ function a_write(parent::Union{File,Object}, name::AbstractString, data; pv...)
     try
         writearray(obj, dtype.id, data)
     catch exc
-        o_delete(obj)
```

Comment:
Thanks, but did it looks like you accidentally delete this? This is still problematic and `o_delete` won't work when `obj` is an Attribute.

Reply:
ops my bad looking at the wrong thing